api/src/paths/administrative-activity/system-access/{administrativeActivityId}/approve.ts

```diff
@@ -18,7 +18,7 @@ export const PUT: Operation = [
     return {
       and: [
         {
-          validSystemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN],
+          validSystemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.DATA_ADMINISTRATOR],
```

Comment:
Just checking: Do we want data admins to be able to do user management stuff? I thought that was the one aspect that was system admin only?

Reply:
Or at least, that was the desire before, unless it has since changed

Reply:
Edit: as discussed, this is the desired behaviour for now.